### PR TITLE
WIP: Doc and unit change

### PR DESCRIPTION
**Pull request by licode**
more doc is required, and unit change on cs is updated.

### CODE REVIEW

File: conda-recipe/meta.yaml [hunk 1]
```diff
@@ -20,7 +20,7 @@ requirements:
     - enaml
     - h5py
     - six
-    - matplotlib ==1.4.3
+    - matplotlib
```
Comment (ericdill) [inline conda-recipe/meta.yaml]: this is unpinned because of the new enaml conda packages>

Comment (licode) [inline conda-recipe/meta.yaml]: yep. no constraint on matplotlib now.

File: pyxrf/model/draw_image.py [hunk 2]
```diff
@@ -402,7 +396,7 @@ def show_image(self):
                     if k in name_not_scalable:
                         data_dict = self.dict_to_plot[k]
                     else:
-                        data_dict = self.dict_to_plot[k]/self.scaler_data*ic_norm
```
Comment (ericdill) [inline pyxrf/model/draw_image.py]: I guess we're now assuming that the data is coming in normalized?

Comment (licode) [inline pyxrf/model/draw_image.py]: correct.

**Comment by ericdill** [comment]
Cool. LGTM. Anything else going into this PR?

**Comment by licode** [comment]
still WIP, need to add something related to getting data from databroker.

**Comment by ericdill** [comment]
Ah ok. I'll add "WIP: " to the title

**Pull request merged**